blog/2020-10-27-hello-debezium.adoc

```diff
@@ -0,0 +1,18 @@
+= Hello Debezium!
+anmohant
+:awestruct-tags: [ community, news ]
+:awestruct-layout: blog-post
+
+Hello everyone, my name is Anisha Mohanty and I recently joined Red Hat and the Debezium team.
+
+I started my journey with Redhat in April 2020 after completing my graduation. I was introduced to open source in my early college days, but I wasn't aware of how organizations work and wanted to get the essence of open source ethics and values. That is something that I am fascinated to learn as I joined Redhat.
+
+My work started under the Data Virtualization team with Teiid and then under the https://graphqlcrud.org/[GRAPHQLCRUD] project which is a standard for the generic query interface on top of GraphQL. The project has started well and is in great shape right now. We have successfully added CRUD capabilities, paging, and filtering specifications.
+
++++<!-- more -->+++
```

Comment:
This controls how much is shown on the blog front page. Perhaps pull it up by one paragraph?

Reply:
Yes. Will be better.